.importjs.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
       "isEqual",
       "isEmpty",
       "every",
-      "kebabCase"
+      "kebabCase",
+      "camelCase"
```

Comment:
what's this addition for?

Reply:
This is because the importjs tool I use to auto-import modules doesn't know about the named exports of lodash. So I've been slowly adding them. This must have been for a different branch that stuck in here.